solr/core/src/test/org/apache/solr/cloud/TestConfigSetsAPI.java

```diff
@@ -1590,8 +1590,7 @@ private long uploadGivenConfigSet(
       Map<?, ?> map =
           postDataAndGetResponse(
               cluster.getSolrClient(),
-              cluster.getJettySolrRunners().get(0).getBaseUrl().toString().replace("/solr", "")
-                  + uriEnding,
+              cluster.getJettySolrRunners().get(0).getBaseURLV2().toString() + uriEnding,
```

Comment:
> Just wondering if it does it implicitly.

It doesn't 😦 

@epugh I agree with what I think is your implicit point (and with a point you made explicitly on one of my recent PRs haha) -  that we should add a variant of these "getUrl" methods that returns a string instead of a URL, since this is such a common idiom.

But let's handle that as a separate PR.